fabric-sponge-command-api-v2/src/main/java/net/fabricmc/fabric/api/command/v2/Location.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.command.v2;
+
+import net.minecraft.util.math.BlockPos;
+
+public class Location<T> {
```

Comment:
Why is this even needed? Couldn't the world and pos be passed as seperate parameters

Reply:
yes but Location is more spongy

Reply:
fine ill crob this too